zhaquirks/tuya/ts0601_illuminance.py

```diff
@@ -0,0 +1,115 @@
+"""Tuya HOME-LUX illuminance sensor."""
+
+import math
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import DPToAttributeMapping, TuyaLocalCluster, TuyaNewManufCluster
+
+TUYA_BRIGHTNESS_LEVEL_ATTR = 0x01  # 0-2 "Low, Medium, High"
+TUYA_ILLUMINANCE_ATTR = 0x02  # [0, 0, 3, 232] illuminance
+
+TUYA_BRIGHTNESS_LEVEL_STR = ["Low", "Medium", "High"]
+
+
+class TuyaIlluminanceMeasurement(IlluminanceMeasurement, TuyaLocalCluster):
+    """Tuya local IlluminanceMeasurement cluster."""
+
+    attributes = IlluminanceMeasurement.attributes.copy()
+    attributes.update(
+        {
+            0xFF00: ("brightness_level", t.CharacterString, True),
+        }
+    )
+
+
+class TuyaIlluminanceCluster(TuyaNewManufCluster):
+    """Tuya Illuminance cluster."""
+
+    dp_to_attribute: Dict[int, DPToAttributeMapping] = {
+        TUYA_BRIGHTNESS_LEVEL_ATTR: DPToAttributeMapping(
+            TuyaIlluminanceMeasurement.ep_attribute,
+            "brightness_level",
+            lambda x: TUYA_BRIGHTNESS_LEVEL_STR[x],
+        ),
+        TUYA_ILLUMINANCE_ATTR: DPToAttributeMapping(
+            TuyaIlluminanceMeasurement.ep_attribute,
+            "measured_value",
+            lambda x: (10000.0 * math.log10(x) + 1.0 if x != 0 else 0),
+        ),
+    }
+
+    data_point_handlers = {
+        TUYA_BRIGHTNESS_LEVEL_ATTR: "_dp_2_attr_update",
+        TUYA_ILLUMINANCE_ATTR: "_dp_2_attr_update",
+    }
```

Comment:
You will need to change the imports statements to the new ones:
```python
from zhaquirks.tuya import TuyaLocalCluster
from zhaquirks.tuya.mcu import (
    DPToAttributeMapping,
    TuyaDPType,
    TuyaMCUCluster,
)
```

Change your `TuyaIlluminanceCluster` class declaration to something like:
```python
class TuyaIlluminanceCluster(TuyaMCUCluster):
    """Tuya Illuminance cluster."""

    dp_to_attribute: Dict[int, DPToAttributeMapping] = {
        TUYA_BRIGHTNESS_LEVEL_ATTR: DPToAttributeMapping(
            TuyaIlluminanceMeasurement.ep_attribute,
            "brightness_level",  # or the new name for attribute
            dp_type=TuyaDPType.ENUM,
            converter=lambda x: BrightnessLevel(x),
        ),
        TUYA_ILLUMINANCE_ATTR: DPToAttributeMapping(
            TuyaIlluminanceMeasurement.ep_attribute,
            "measured_value",
            dp_type=TuyaDPType.VALUE,
            converter=lambda x: (10000.0 * math.log10(x) + 1.0 if x != 0 else 0),
        ),
    }

    data_point_handlers = {
        TUYA_BRIGHTNESS_LEVEL_ATTR: "_dp_2_attr_update",
        TUYA_ILLUMINANCE_ATTR: "_dp_2_attr_update",
    }
```

Thats all.